newrelic.js

```diff
@@ -15,7 +15,15 @@ if(config.core.newrelic.key) exports.config = {
 	},
 	rules: {
 		ignore: [
-			'^/socket/\\*',
+			'^/engine.io',
+		],
+		name: [
+			{ pattern: "/me",   name: "/me"  },
+			{ pattern: "/i/.*", name: "/i/*" },
+			{ pattern: "/s/.*", name: "/s/*" },
+			{ pattern: "/r/.*", name: "/r/*" },
+			{ pattern: "/[\w\-]*/.*", name: "/:room/:discussion" },
+			{ pattern: "/[\w\-]*(/?|\?|$)", name: "/:room" }
```

Comment:
The room and discussion regexes seem to be wrong, it is not working on HeyNeighbor. We need to test and correct it first.

Also, we should add nginx config to serve /s/* URLs and favicon.ico directly without hitting scrollback.